apps/backend/src/main.ts

```diff
@@ -36,6 +39,15 @@ async function bootstrap() {
     })
   );
   app.use(json({limit: '50mb'}));
+  app.use(
+    session({
+      secret: configService.get('JWT_SECRET') || generateDefault(),
+      saveUninitialized: true,
```

Comment:
According to the docs

> Forces a session that is "uninitialized" to be saved to the store. A session is uninitialized when it is new but not modified. **Choosing false is useful for implementing login sessions**, reducing server storage usage, or complying with laws that require permission before setting a cookie. Choosing false will also help with race conditions where a client makes multiple parallel requests without a session.

Is it possible to set this to false or does the app not work correctly then?

Reply:
Both of these work when false